_monitoring-your-cluster/metrics/getting-started.md

```diff
@@ -0,0 +1,97 @@
+
+
+## Getting started
+
+The Metrics Framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently using the metrics framework feature flag. 
+
+Enabling this feature can consume system resources. Before enabling the metrics framework, consider if you have enough cluster resources to allocate.
+{: .warning}
+
+### Enabling the feature flag on a node using tarball
+
+The `enable` flag is toggled using a new Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
+
+#### Option 1: Enable the experimental feature flag in the `opensearch.yml` file
+
+1. Navigate to your OpenSearch directory using the following command:
+
+  ```bash
+  cd \path\to\opensearch
+  ```
+
+2. Open your the `opensearch.yaml` file.
+3. Add the following setting to `opensearch.yaml`:
+
+  ```bash
+  opensearch.experimental.feature.telemetry.enabled=true
+  ```
+  {% include copy.html %}
+
+4. Save your changes and close the file.
+
+#### Option 2: Modify jvm.options
+
+To enable the metrics framework feature using `jvm`, add the following line to `config/jvm.options` before starting the OpenSearch:
+
+```bash
+-Dopensearch.experimental.feature.telemetry.enabled=true
+```
+{% include copy.html %}
+
+#### Option 3: Enable from an environment variable
+
+You can enable the metrics framework in a single command by adding the metrics framework environment variable to the `OPENSEARCH_JAVA_OPTS` command, as shown in the following example:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true" ./opensearch-2.9.0/bin/opensearch
+```
+{% include copy.html %}
+
+You can also define the environment variable separately before running OpenSearch by running the following command:
+
+```bash
+export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+ ./bin/opensearch
+```
+{% include copy.html %}
+
+### Enable with Docker containers
+
+If you’re running OpenSearch using Docker, add the following line to `docker-compose.yml` under `environment`:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+```
+{% include copy.html %}
+
+
+### Enable metrics framework
+
+Once you've enabled the feature flag, you can enable the metrics framework by using the following setting that enables metrics in the opensearch.yaml:
+
+```bash
+telemetry.feature.metrics.enabled=true
+```
+
+The metrics framework supports various telemetry solutions through plugins. Use the following instructions to enable the `telemetry-otel` plugin:
```

Comment:
The two items below do not read as "instructions" (we don't tell the reader to do anything).